protocol/x/revshare/types/errors.go

```diff
@@ -36,9 +36,9 @@ var (
 		6,
 		"total fees shared exceeds net fees",
 	)
-	ErrAffiliateFeesSharedExceedsNetFees = errorsmod.Register(
+	ErrAffiliateFeesSharedGreaterThanOrEqualToNetFees = errorsmod.Register(
 		ModuleName,
 		7,
-		"affiliate fees shared exceeds net fees",
+		"net fees minus affiliate fee share is not larger than zero",
```

Comment:
Nit: var name and err message is saying the same thing but differently:

```
AffiliateFee >= netFee
netFee - AffiliateFee <= 0
```

Let's make them consistent for readability